src/server/session.ts

```diff
@@ -736,11 +749,10 @@ namespace ts.server {
                     highlightSpans: highlightSpans.map(convertHighlightSpan)
                 };
 
-                function convertHighlightSpan(highlightSpan: HighlightSpan): protocol.HighlightSpan {
+                function convertHighlightSpan(this: Session, highlightSpan: HighlightSpan): protocol.HighlightSpan {

```

Comment:
not sure i understand why you need the `this` parameter here...

Reply:
This is an error, rollback.